ansible_base/rbac/models.py

```diff
@@ -30,9 +30,13 @@
 class DABPermission(models.Model):
     "This is a minimal copy of auth.Permission for internal use"
 
-    name = models.CharField("name", max_length=255)
-    content_type = models.ForeignKey(ContentType, models.CASCADE, verbose_name="content type")
-    codename = models.CharField("codename", max_length=100)
+    name = models.CharField("name", max_length=255, help_text=_("The name of this permission."))
+    content_type = models.ForeignKey(ContentType, models.CASCADE, verbose_name="content type", help_text=_("The content type this permission will apply to."))
+    codename = models.CharField(
+        "codename",
+        max_length=100,
+        help_text=_("The name of the permission, giving the action and the model, from the Django Permission model."),
```

Comment:
This is the only one that matters, and unfortunately, you can't say the format rule _always_ applies, but this does conform to the existing Django auth app pattern.

So the format of the permission is usually in the form of `{action}_{model_name}`, but can be customized for a specific entry in a model's Meta class. The model name is well-defined formally, and the action is usually / probably the DRF viewset action. This is actually under-specified and in some internal Django uses the actually unique combination of `(app_label, model_name, action)` is used. But in DAB RBAC, it is expected to be unique without app_label, due to additional enforcement.

Reply:
How does this sound?
```
The name of the permission. This is in the format {action}_{model_name}. Where action is typically the view set action (view/list/etc) from Django rest framework. 
```

Reply:
I don't think we would want to call this `The name of the permission....` because that is the text used for `name` on line 33.
As to the comment about the format, we should only include the format if the model class itself sets format.

Reply:
@tznamena Good callout